src/recs.js

```diff
@@ -232,9 +245,10 @@ export async function syncTemplate(options) {
 	}
 }
 
-export function generateTemplateSettings({ name, description }) {
+export function generateTemplateSettings({ name, description, type }) {
 	const settings = {
 		type: TEMPLATE_TYPE_RECS,
+		isEmail: Boolean(type === 'email'),
```

Comment:
What do you think about leveraging the `type` property for this instead of creating a new one (`isEmail`) ?

Currently we have:
```
const TEMPLATE_TYPE_RECS = 'snap/recommendation';
```

Maybe we could do something like:
```
const settings = {
  type: `${TEMPLATE_TYPE_RECS}/${type}`
}
```

This would of course affect the `buildTemplatePayload` function slightly...